changelogs/archive_fragments/2.2.0/v2.2.0_summary.yaml

```diff
@@ -0,0 +1,15 @@
+release_summary: |
+        | Release Date: 2023-0531
+        | This release provides Ansible compliance changes required on top of the changes included in
+        | the 2.1.0 release of the enterprise_sonic Ansible network resource module collection.
+        | It addresses two issues raised by the Ansible core team with the content of the 2.1.0 release.
+        | 1) Back out the "breaking_change" made in the sonic_aaa resource module to to fix a functional
```

Comment:
```suggestion
        | 1) Back out the "breaking_change" made in the sonic_aaa resource module to fix a functional
```

Reply:
I manually edited this before seeing this comment.

Reply:
Fragment files are not used after they are incorporated into the changelog.yaml file and archived. The CHANGELOG.rst file is re-generated based solely on the changelog.yaml file and does not refer to archived fragments.

I am accepting this change to resolve the issue, but in the future, it is not necessary to check or correct archived fragment files when reviewing release notes.